docs/changelog/3_x.rst

```diff
@@ -0,0 +1,726 @@
+====
```

Comment:
Can you mark it as a "new in 3.0" kind of page?

Reply:
Because it's not yet released

Reply:
Is this what you intended, @1st1?
<img width="771" alt="image" src="https://user-images.githubusercontent.com/2027711/236560209-4216357a-8fac-4f60-b627-717665094763.png">

I've done it, but it's a little weird since it now responds to the version selector even though it's outside the versioned fence.

Reply:
Got it. Let's just change the title of the page to `EdgeDB v3 (dev)` and the TOC entry to `v3.0 (dev)`? (keeping the filename as is)